frontend/src/components/Layout/MobileSearch/MobileSearchContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+import { searchAllCourses } from 'api/courses';
+import { Course } from 'types/course';
+
+import MobileSearch from './MobileSearch';
+
+const MobileSearchContainer: React.FC = () => {
+  const [isSearchOpen, setSearchOpen] = useState<boolean>(false);
+  const [coursesFound, setFoundCourses] = useState<Array<Course>>([]);
```

Comment:
Maybe we should rename useState variables to `const [coursesFound, setCoursesFound]`